Module/client/src/components/app.jsx

```diff
@@ -17,25 +18,49 @@ class App extends React.Component {
     this.getData();
   }
 
+  // initial and only request to server for mockData
   getData() {
     axios.get('/listinginfo').then((response) => {
       let { id } = this.state;
-      if (!id || (id > 100) || (id < 0)) { id = 0; }
+      // sets id to the first mock data example if id is undefined or outside range
+      if (!id || (id > 101) || (id < 0)) { id = 0; }
       this.setState({ data: response.data[id], loading: false });
     });
   }
 
   render() {
     const { loading, data } = this.state;
     if (loading) return null;
-    const { hostName, header, neighborhood } = data;
+    // deconstructing data that will be sent down as props
+    const {
+      hostName, homeType, header, neighborhood, bathrooms, bedrooms, capacity,
+      beds, primTrait, secondTrait, tertTrait,
+    } = data;
+    // similar to Airbnb, this sets the description as tab title
```

Comment:
This is well done, I hadn't noticed that they do that.

Reply:
Thanks!